cli_client/python/timesketch_cli_client/commands/timelines.py

```diff
@@ -92,3 +92,36 @@ def event_add_tags(ctx, timeline_id, event_id, tags, output):
         click.echo(return_value)
     else:
         click.echo(f"Tags {tags_list} added to event {event_id}")
+
+
+@timelines_group.command("event-add-comment")
```

Comment:
I find the event-add-comment not fitting in to the UI. I wonder if we want to create a subgroup under timeline.

Reply:
WDYT about:
```
$timesketch timeline 19 event 1234 --comment ...`
$timesketch timeline 19 event 1234 --tag tag2,tag2 ...`
```

Reply:
chatted offline, we will move that to a specific events click group instead